worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -252,6 +252,18 @@ private void validateTaskMessage(TaskMessage tm) throws InvalidTaskException
 
         private boolean isTaskIntendedForThisWorker(final TaskMessage tm, final TaskInformation taskInformation)
         {
+            if (!Boolean.parseBoolean(
+                    System.getProperty("CAF_WORKER_ENABLE_DIVERTED_TASK_CHECKING", "true"))) {
+
+                LOG.debug(
+                    "Diverted task checking is disabled, so assuming that Task {} (message id: {}) on input queue {} "
+                            + "is intended for this worker",
+                    tm.getTaskId(),
+                    taskInformation.getInboundMessageId(),
+                    workerQueue.getInputQueue());                
+                
```

Comment:
Remove the extra blank lines at the end of the line:

```suggestion
                    workerQueue.getInputQueue());

```